packages/next-templates/src/app/page.tsx

```diff
@@ -32,21 +36,36 @@ export default function Home() {
       <ExampleNavigation />
       <PageContent className="voorbeeld-page-content--flex">
         <section className="voorbeeld-section">
-          <TopTaskNavigation className="grid">
-            <TopTaskLink icon={<UtrechtIconPaspoort />}>Paspoort of ID-kaart aanvragen</TopTaskLink>
-            <TopTaskLink href="/meldingen/" icon={<UtrechtIconMeldingKlacht />}>
-              Meldingen openbare ruimte
-            </TopTaskLink>
-            <TopTaskLink icon={<UtrechtIconVerhuizen />}>Verhuizing doorgeven</TopTaskLink>
-            <TopTaskLink icon={<UtrechtIconWerken />}>Werken bij de gemeente</TopTaskLink>
-            <TopTaskLink icon={<UtrechtIconNummerbord />}>Parkeren: kentekenwijziging doorgeven</TopTaskLink>
-            <TopTaskLink icon={<UtrechtIconAfvalScheiden />}>Afval</TopTaskLink>
-          </TopTaskNavigation>
+          <div className="voorbeeld-section-content voorbeeld-toptask-layout">
+            <TopTaskNavigation className="grid">
+              <TopTaskLink icon={<UtrechtIconPaspoort />}>Paspoort of ID-kaart aanvragen</TopTaskLink>
+              <TopTaskLink href="/meldingen/" icon={<UtrechtIconMeldingKlacht />}>
+                Meldingen openbare ruimte
+              </TopTaskLink>
+              <TopTaskLink icon={<UtrechtIconVerhuizen />}>Verhuizing doorgeven</TopTaskLink>
+              <TopTaskLink icon={<UtrechtIconWerken />}>Werken bij de gemeente</TopTaskLink>
+              <TopTaskLink icon={<UtrechtIconNummerbord />}>Parkeren: kentekenwijziging doorgeven</TopTaskLink>
+              <TopTaskLink icon={<UtrechtIconAfvalScheiden />}>Afval</TopTaskLink>
+            </TopTaskNavigation>
+            <TopTaskHeader>
```

Comment:
Ik vraag me af of de naam van deze component goed is gekozen. Ik denk niet dat het iets te maken heeft met een header. Zou het niet een TopTaskCard zijn?